.github/workflows/docs.yml

```diff
@@ -114,18 +126,32 @@ jobs:
         with:
           repository: 'opendp/artifacts'
           path: ./artifacts
-      
+
       - name: Populate docs site with pdfs
         run: |
+          # get the hash of the stable branch
+          stable_hash=$(git rev-parse origin/stable 2>/dev/null || true)
+
           # copy each folder over
           for folder in en/*; do
-            source=artifacts/release/$(basename $folder)
+            release=$(basename $folder)
+            source=artifacts/release/$release
             target=$folder/proofs
 
             if [ -d "$source" ]; then
-              mkdir -p "$target";
-              mv "$source"/* "$target";
-            fi;
+              echo "moving $source/* to $target"
+              mkdir -p "$target"
+              mv "$source"/* "$target"
+
+              # if this release is the stable branch, duplicate it
+              release_hash=$(git rev-parse $release 2>/dev/null || true)
+              if [[ "$release_hash" == "$stable_hash" ]]; then
+                stable_target=en/stable/proofs
+                echo "copying $target/* to $stable_target"
+                mkdir -p "$stable_target"
+                cp -rp "$target"/* "$stable_target"
+              fi
```

Comment:
I thought we weren't going to duplicate this? Any links on stable will naturally point to a versioned location. Thus there are no /stable links on the docs site.

Reply:
You're right, don't know why I left this in, have removed it.